README.md

```diff
@@ -53,7 +53,7 @@ Please check the [roadmap](https://github.com/AzureAD/microsoft-authentication-l
 
 -   [GitHub Issues](../../issues) is the best place to ask questions, report bugs, and new request features.
 
--   [FAQs](https://github.com/AzureAD/microsoft-authentication-library-for-js/wiki/FAQs) for access to our frequently asked questions.
+-   [FAQs](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/FAQ.md) for access to our frequently asked questions.
```

Comment:
why don't we create a aka.ms link for this ?

Reply:
> why don't we create a aka.ms link for this ?

@ejahja Might be good for you to own such a link from an administrative standpoint ^^

Reply:
https://aka.ms/msaljs-faq